workers/Cargo.lock

```diff
@@ -0,0 +1,3581 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Delete these lockfiles please, it should be in `members` in the top level cargo manifest